webservices/resources/decorators.py

```diff
@@ -0,0 +1,36 @@
+from functools import wraps
+from time import time
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+def timing(func):
```

Comment:
Is this used?

Reply:
I only used it for performance testing purpose. I'll remove it for now.